src/app/qan/profile-details/components/details-row/details-row.component.html

```diff
@@ -22,7 +22,7 @@
       {{isSum ? (currentMetric?.stats?.sum | humanize: pipeInfo?.sumPipe) : '0' }}
     </span>
     <span *ngIf="percentOfTotal" class="total-percent">
-      {{ percentOfTotal | humanize: 'percent' }} of total
+      {{ percentOfTotal }} % of total
```

Comment:
```suggestion
      {{ percentOfTotal }}% of total
```
Please remove space before '%'.